trio/_socket.py

```diff
@@ -241,9 +292,14 @@ def from_stdlib_socket(sock):
 
 
 @_wraps(_stdlib_socket.fromfd, assigned=(), updated=())
-def fromfd(fd, family, type, proto=0):
+def fromfd(
+    fd: SupportsIndex,
+    family: AddressFamily | int = _stdlib_socket.AF_INET,
+    type: SocketKind | int = _stdlib_socket.SOCK_STREAM,
```

Comment:
Were the defaults missing before?

Reply:
yep, they were forwarded with `getattr()` and didn't have it's parameters typed out at all.